.gitlab/ci/on-push.yml

```diff
@@ -262,3 +262,20 @@ slack-notify-nightly-build:
     # when triggering pipeline with alternate env variable value passed in the API call
     SLACK_CHANNEL: $SLACK_CHANNEL
     SLACK_JOB: 'true'
+
+
+eyal-test:
+  tags:
+    - gke
+  extends:
+    - .default-job-settings
+  variables:
+    ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_XSOAR}"
+    KUBERNETES_CPU_REQUEST: 1000m
```

Comment:
are we sure that this is necessary here? (we should avoid using this in a regular basis)

Reply:
@esharf its for testing proposes only for now. It wont be part of the final PR.